### PR TITLE
Dev setup

### PR DESCRIPTION
**Pull request by viv9k**
### Functionality:
Added progress Indicators

### Solution:
Added Progress indicator in setup component

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
Tested while setup

### CODE REVIEW

File: src/app/body/task-details/task-details.component.html [hunk 1]
```diff
@@ -79,25 +79,19 @@ <h2>Task Details</h2>
             <div class="col-md-6 part">
                 <div class="row">
                     <div class="col-2 creationDate">
+                        <span id="headerLabel" style="cursor: default;">Creation Date</span>
                         <h5 class="pt-1">{{task.CreationDate}}</h5>
                     </div>
                     <div class="col-4 assignee">
                         <span id="headerLabel" style="cursor: default;">Assignee</span>
                         <app-suggestion-bucket [email]="task.Assignee" (selectedEmail)="selectedAssignee($event)">
                         </app-suggestion-bucket>
```
Comment (arjund-06) [inline src/app/body/task-details/task-details.component.html:88]: The creator name and profile is not showing and the profile pics of users in the activity area too isnt coming

**Review by arjund-06 — CHANGES_REQUESTED** [review]

File: functions/model/users/tark/getUserByUid.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        });
+
+        Promises.push(promise);
+
+        if (uids.length > 10) {
+            uids.splice(0, 10);
+        } else {
+            uids.splice(0, uids.length);
+        }
+    }
```
[the remark this answers is not in the document]
Comment (viv9k) [reply functions/model/users/tark/getUserByUid.js:47]: yeah they are part of the login to handle it better

File: src/app/body/task-details/task-details.component.html (hunk 1 above)
Comment (viv9k) [reply src/app/body/task-details/task-details.component.html:88]: Yeah I will check it

Comment (viv9k) [reply src/app/body/task-details/task-details.component.html:88]: But it should be resolved once the api is deployed

**Review by arjund-06 — APPROVED** [review]
ship it

**Pull request merged**